core/src/main/java/org/infinispan/commands/triangle/BackupWriteCommand.java

```diff
@@ -46,7 +46,8 @@
    public final CompletionStage<?> invokeAsync(ComponentRegistry componentRegistry) {
       WriteCommand command = createWriteCommand();
       command.init(componentRegistry);
-      command.setFlagsBitSet(flags);
+      // Mark the command as a backup write
+      command.setFlagsBitSet(flags | FlagBitSets.BACKUP_WRITE);
       command.addFlags(FlagBitSets.SKIP_LOCKING);
```

Comment:
nit: can you merge `FlagBitSets.SKIP_LOCKING` to the line above and remove this line?

Reply:
Ah yes, didn't notice 👍